New Assignment Page: make input fields consistent

yummypasta opened the pull request:
### Fixes #555 

#### Describe changes made in this PR
I changed the width and height of the `Name` and `Deadline` input fields in the New Assignment page to visually match the styling of the `Grading scale` input field.

### Screenshots
<img width="1430" alt="Screen Shot 2020-01-01 at 9 40 11 PM" src="https://user-images.githubusercontent.com/12377772/71653094-7033e600-2cdf-11ea-9942-6ae927fad999.png">

<!--
*** OLD IMAGE *** (before UI changes)
<img width="867" alt="Screen Shot 2019-12-31 at 12 44 21 PM" src="https://user-images.githubusercontent.com/12377772/71633783-47bac780-2bcb-11ea-9b93-0aaa86abd273.png">
***
-->

yummypasta commented:
I actually could not test this after merging `CircuitVerse/master` into my branch. With the new UI, none of the navbar buttons work, I could not go to the sign in page, and I could not view my groups page.

Is there a way I can test this? It theoretically should work, but I don't know if my CSS changes translate over to SCSS 100%.

Ananthavijay commented:
@yummypasta  make the changes locally and test it out.. Don't include the navigation problem in this PR as it has already been added as an beginner task

aayush-05 commented:
@yummypasta use writing urls in the search bar to get to a certain page, the route links are all same. For example, use localhost:3000/users/sign_in to get into the login window section and move forward.
Or you can try gitpod as an easy way, but I am not sure if the login works in that.

yummypasta commented:
@aayush-05 I've tried using direct URLs, but they either redirect (such as `/users/sign_in`) back to the home page, or they send the error:
```
The record you wish access could not be found
```
Is there a workaround?

aayush-05 commented:
It is working fine for me
![image](https://user-images.githubusercontent.com/47032027/71652791-1a4a5700-2d4e-11ea-9954-5177519b2fa2.png)

Please check that you are running the latest commit from the master. Also check the comment I made on the GCI dashboard

yummypasta commented:
@aayush-05 Sorry, I just had leftover cookies and I didn't realize that the "login" button gets replaced with "notifications." I've updated my changes to be consistent with #623. I've updated the screenshots on the original comment.

JoshHeng commented on app/assets/stylesheets/layout.scss:
```diff
@@ -13,6 +13,11 @@
   height: 25px;
 }
 
+#assignment_name, #party {
```
Either use inline styling or a class instead

JoshHeng commented on app/assets/stylesheets/layout.scss:
```diff
@@ -13,6 +13,11 @@
   height: 25px;
 }
 
+#assignment_name, #party {
```
Either use inline styling or a class instead

MrBartusek commented:
#778 has been merged and #555 has been closed so this PR should be closed too
